lib/elements/elements.css

```diff
@@ -216,8 +216,66 @@
   top: 72px;
 }
 
+.modal {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: rgba(70,70,70,0.75);
+  z-index: 99999;
+  transition: opacity 400ms ease-in;
+}
+
+.modal > div {
+  width: 400px;
+  position: relative;
+  margin: 10% auto;
+  padding: 5px 20px 13px 20px;
+  border-radius: 4px;
+  box-shadow: 0 0 18px rgba(0,0,0,0.4);
```

Comment:
I'm hoping that some of the css styles I checked in for the `.dialog` will be helpful here.